_locales/en/messages.json

```
@@ -403,10 +403,10 @@
     "message": "This profile was deleted"
   },
   "LabelNoAccount": {
-    "message": "No profiles here"
+    "message": "No profiles yet"
   },
   "DescriptionNoAccount": {
-    "message": "Create a new profile to sync your bookmarks or import profiles from a different device or browser."
+    "message": "Add new & existing profiles, or Import from a file, then you can begin syncing."
```

Thanks for the feedback! I agree that this is slightly confusing.
What would be the difference between adding an existing profile and importing?

---

My understanding:
- Importing is from a file.
- Profile creation and accessing of existing account + their profiles is through logging into the webui.

---

It could be clearer to say:
`import file`

---

```suggestion
    "message": "Add new profiles, or import profiles from a file, then you can begin syncing."
```

I think this is a bit clearer as "add existing profiles" sounds like an oxymoron to me ;)

---

I think the only word slightly tripping me up now is "new".

Thank you for accepting! :heart: